src/hooks/useEnabledExtensions.tsx

```diff
@@ -2,23 +2,21 @@ import { Extension } from '@colony/colony-js';
 
 import useExtensionsData from './useExtensionsData';
 
-type EnabledExtensionKey = `is${Extension}Enabled`;
-type EnabledExtensions = Partial<Record<EnabledExtensionKey, boolean>>;
-
 const useEnabledExtensions = () => {
   const { installedExtensionsData, loading } = useExtensionsData();
 
-  const enabledExtensions = installedExtensionsData.reduce<EnabledExtensions>(
-    (extensions, extension) => ({
-      ...extensions,
-      [`is${extension.extensionId}Enabled`]: extension.isEnabled,
-    }),
-    {},
```

Comment:
Keeping the dynamic reduce approach would mean I still have to search for the VotingReputation installation separately. Thinking about it, we don't add new extensions every day so I think it makes sense to be fixed for those extensions we care about